doc/build-osx.md

```diff
@@ -80,9 +80,9 @@ Running
 
 Bitcoin Core is now available at `./src/bitcoind`
 
-Before running, it's recommended that you create an RPC configuration file:
+Before running, you may create an empty RPC configuration file:
```

Comment:
Do we generally refer to `bitcoin.conf` as "configuration file" or "RPC configuration file"?